packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,540 @@
+			}
+		]
+	},
+	"button_functionality": {
+		"label": "Button",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Toggle",
+				"value": 0
+			},
+			{
+				"label": "Turn On, Dim Up",
+				"value": 1
+			},
+			{
+				"label": "Turn Off, Dim Down",
+				"value": 2
+			},
+			{
+				"label": "Momentary",
+				"value": 3
+			}
+		]
+	},
+	"global_brightness": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "LED Indicator Brightness:",
+		"valueSize": 4,
+		"defaultValue": 255
+	},
+	"color_command_button": {
+		"label": "Color Commands LED Indicator",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Direct Control",
+				"value": 0
+			},
+			{
+				"label": "Color For OFF State",
+				"value": 1
+			},
+			{
+				"label": "Color For ON State",
+				"value": 2
+			}
+		]
+	},
+	"dimming_duration": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration Of Dimming",
+		"unit": "seconds",
+		"defaultValue": 5
+	},
+	"duration_on_off": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration Of On/Off",
+		"defaultValue": 0
+	},
+	"dimmer_mode": {
+		"label": "Dimmer Mode",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "No dimming, Switch Only",
+				"value": 0
+			},
+			{
+				"label": "Trailing Edge",
+				"value": 1
+			},
+			{
+				"label": "Leading Edge",
+				"value": 2
+			}
+		]
+	},
+	"dimmer_min": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer Minimum Level",
+		"defaultValue": 0
+	},
+	"dimmer_max": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Dimmer Maximum Level",
+		"defaultValue": 99
+	},
+	"dimmer_metering": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Meter Report Time",
+		"defaultValue": 60
+	},
+	"led_indicator": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Button LED Indicator",
+		"unsigned": true,
+		"valueSize": 4,
+		"defaultValue": 0
+	},
+	"button_led_indicator": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Button LED Indicator",
+		"maxValue": 7,
+		"defaultValue": 7,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled, External Control only",
+				"value": 0
+			},
+			{
+				"label": "Follow Switch",
+				"value": 1
+			},
+			{
+				"label": "Follow Switch, Inverted",
+				"value": 2
```

Comment:
Why do these options repeat? Are they different somehow?

Reply:
No these are the same , this is just defined in the manual as ‘same as 1’ and ‘same as 2’

Reply:
Weird. Let's just omit the duplicate options but leave the value range in case it's already set to one of those when it's included.

Reply:
Hmm alright give me 15 min and ill make the change